src/main/java/gregtech/common/metatileentities/multi/MetaTileEntityFissionReactor.java

```diff
@@ -0,0 +1,895 @@
+                    i))
+                break;
+            i++;
+        }
+        return i;
+    }
+
+    /**
+     * Checks for casings on top or bottom of the controller to determine the height of the reactor
+     */
+    public int findHeight(boolean top) {
+        int i = 1;
+        while (i <= 15) {
+            if (this.isBlockEdge(this.getWorld(), this.getPos(), top ? EnumFacing.UP : EnumFacing.DOWN, i))
+                break;
+            i++;
+        }
+        return i - 1;
+    }
+
+    @Override
+    public void updateFormedValid() {
+        // Take in coolant, take in fuel, update reactor, output steam
+
+        if (!this.getWorld().isRemote && this.getOffsetTimer() % 20 == 0) {
+            if (this.lockingState == LockingState.LOCKED) {
+                // Coolant handling
+                this.fissionReactor.makeCoolantFlow(flowRate);
+
+                // Fuel handling
+                if (this.fissionReactor.fuelDepletion == 1.) {
+                    boolean canWork = true;
+                    for (IFuelRodHandler fuelImport : this.getAbilities(MultiblockAbility.IMPORT_FUEL_ROD)) {
+                        canWork &= !fuelImport.getStackHandler().extractItem(0, 1, true).isEmpty();
+                        // We still need to know if the output is blocked, even if the recipe doesn't start yet
+                        canWork &= ((MetaTileEntityFuelRodImportHatch) fuelImport).getExportHatch(this.height - 1)
+                                .getExportItems().insertItem(0,
+                                        OreDictUnifier.get(OrePrefix.fuelRodHotDepleted, fuelImport.getFuel()), true)
+                                .isEmpty();
+                    }
+
+                    for (IFuelRodHandler fuelImport : this.getAbilities(MultiblockAbility.IMPORT_FUEL_ROD)) {
+                        if (fissionReactor.needsOutput) {
+                            ((MetaTileEntityFuelRodImportHatch) fuelImport).getExportHatch(this.height - 1)
+                                    .getExportItems().insertItem(0,
+                                            OreDictUnifier.get(OrePrefix.fuelRodHotDepleted, fuelImport.getFuel()),
+                                            false);
+                        }
+                        if (canWork) {
+                            fuelImport.getStackHandler().extractItem(0, 1, false);
+                            this.fissionReactor.fuelMass += 60;
+                            fissionReactor.needsOutput = true;
+                        }
+                    }
+
+                    this.fissionReactor.fuelDepletion = 0.;
+
+                }
+            }
+            this.updateReactorState();
+
+            this.syncReactorStats();
+            if (this.fissionReactor.checkForMeltdown()) {
+                SoundManager.getInstance().startTileSound(GTSoundEvents.SUS_RECORD.getSoundName(), 1, this.getPos());
```

Comment:
change this to a different sound

Reply:
Ah, you don't like my debug sounds? But yes, I'll remove it.